properties.schema

```diff
@@ -88,6 +88,15 @@
                   "validators": [],
                   "help": "Your default options here are: icon-globe, icon-language-1, icon-language-2"
                 },
+                "_restoreStateOnLanguageChange": {
+                  "type": "boolean",
+                  "required": true,
+                  "default": true,
+                  "title": "Restore course state.",
+                  "inputType": "Checkbox",
+                  "validators": [],
+                  "help": "Restore course state on language switch."
```

Comment:
I think the `help` text should give some mention that it's strongly recommended that the course structure matches among languages, e.g. for future proofing the authoring tool, something like:
`NOTE: It's recommended that courses share the same structure.`